plugins/bulk-import/src/components/AddRepositories/AddRepositoriesForm.tsx

```diff
@@ -1,16 +1,18 @@
 import React from 'react';
 
+import { WarningPanel } from '@backstage/core-components';
+
 import { makeStyles } from '@material-ui/core';
-import HelpIcon from '@mui/icons-material/HelpOutline';
 import FormControl from '@mui/material/FormControl';
-import FormControlLabel from '@mui/material/FormControlLabel';
-import Radio from '@mui/material/Radio';
-import RadioGroup from '@mui/material/RadioGroup';
-import Tooltip from '@mui/material/Tooltip';
-import Typography from '@mui/material/Typography';
-import { useFormikContext } from 'formik';
 
-import { AddRepositoriesFormValues } from '../../types';
+// import HelpIcon from '@mui/icons-material/HelpOutline';
+// import FormControlLabel from '@mui/material/FormControlLabel';
+// import Radio from '@mui/material/Radio';
+// import RadioGroup from '@mui/material/RadioGroup';
+// import Tooltip from '@mui/material/Tooltip';
+// import Typography from '@mui/material/Typography';
+// import { useFormikContext } from 'formik';
+// import { AddRepositoriesFormValues } from '../../types';
```

Comment:
remove commented code block

Reply:
I deliberately didn't remove this code. We aren't supporting ServiceNow approval in this release, but we plan to in the next one. To save extra effort later, I commented it out instead of removing it entirely.